utils/ExampleRunner/template-all-examples.html

```diff
@@ -7,6 +7,31 @@
       content="width=device-width, height=device-height, initial-scale=1, maximum-scale=1, minimum-scale=1, user-scalable=no"
     />
   </head>
+  <script>
+    const browserImportsMap = new Map();
+
+window.browserImportFunction = async (moduleId, options) => {
```

Comment:
And all the templates html now need to add this function should they be wanting to perform lazy imports.
This needs documentation, although I'm not quite sure where yet.